fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/layers/interning/ScopedInterningLayer.java

```diff
@@ -126,6 +127,25 @@ public static ScopedInterningLayer global(@Nonnull FDBDatabase database) {
                 .whenComplete((ignored, th) -> context.close());
     }
 
+    @Override
+    @VisibleForTesting
+    protected CompletableFuture<Void> deleteReverseForTesting(FDBRecordContext context, final long value) {
+        return interningLayerFuture
+                .thenApply(layer -> {
+                    layer.deleteReverseForTesting(context, value);
+                    return null;
+                });
```

Comment:
```suggestion
                .thenAccept(layer -> layer.deleteReverseForTesting(context, value));
```

Reply:
Hmm, that's strange. `.thenAccept` takes a consumer, so a `void` lambda should be able to be automatically translated into a `Void`. But this is minor, so I guess it's not super important.